app/test_split_integer.py

```diff
@@ -2,20 +2,27 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    actual = split_integer(6, 2)
+    assert sum(actual) == 6
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 10
+    number_of_parts = 2
+    actual = split_integer(value, number_of_parts)
+    assert actual == [5, 5]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    actual = split_integer(8, 1)
+    assert actual[0] == 8
```

Comment:
It would be better not to refer to the first element of the list in the assertion clause but rather assert the list as a whole. It will check that there are no excessive elements in the list as well.